src/main/java/lotto/domain/WinningLottoAmount.java

```diff
@@ -1,12 +1,16 @@
 package lotto.domain;
 
-import java.util.*;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
 
 public enum WinningLottoAmount {
 
     FIFTH(3, 5000),
     FOURTH(4, 50000),
     THIRD(5, 1500000),
+    SECOND(5, 30000000),
     FIRST(6, 2000000000),
```

Comment:
언더바 표기법을 이용하면 가독성이 좋아집니다 :)

```suggestion
     FIRST(6, 2_000_000_000),
```